irohad/validation/impl/stateful_validator_impl.cpp

```diff
@@ -18,9 +18,9 @@
 #include <numeric>
 #include <set>
 
-#include "backend/protobuf/from_old_model.hpp"
+#include <boost/range/adaptor/transformed.hpp>
+
```

Comment:
Is the space needed? Maybe we can remove extra spaces between includes

Reply:
yes, it is style guide we following (https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes)